src/main/java/de/qabel/desktop/storage/AbstractNavigation.java

```diff
@@ -67,7 +81,10 @@ public synchronized AbstractNavigation navigate(BoxFolder target) throws QblStor
 			if (cryptoUtils.decryptFileAuthenticatedSymmetricAndValidateTag(indexDl, tmp, key)) {
 				DirectoryMetadata dm = DirectoryMetadata.openDatabase(
 						tmp, deviceId, target.ref, this.dm.getTempDir());
-				return new FolderNavigation(prefix, dm, keyPair, target.key, deviceId, readBackend, writeBackend, getIndexNavigation());
+				FolderNavigation folderNavigation = new FolderNavigation(prefix, dm, keyPair, target.key, deviceId, readBackend, writeBackend, getIndexNavigation());
```

Comment:
temporarily ok, but we have to talk about those n-parameter constructors once this stuff is moved to the core (or at least as much of it as possible) ...